concepts/05 UI Components/Button/00 Getting Started with Button/12 Add an Icon.md

```diff
@@ -0,0 +1,52 @@
+Apart from text, buttons can also display [icons from different sources](/Documentation/Guide/Themes_and_Styles/Icons/). To add an icon, specify the [icon](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#icon) property (applies to all types except *"back"*). In this tutorial, we will use the [DevExtreme icon library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library):
```

Comment:
I don't want to mention "apart from text" because it complicates the sentence and sounds quite obvious. You can display it in addition to text, or no text and only icon, etc. Also wanted to try and not repeat the word "icon" so many times - use a couple of synonyms. 

Also, I recommend to reserve the word "we" for cases when we refer to ourselves, the DevExpress company. "We recommend", "we implemented", "we will show you how"... but "you will do X". Not a big deal, because I know other sources do this... Just makes more sense to me. 
 
```suggestion
The [icon](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#icon) property adds a glyph to the button. (This property is not in effect if a button's type is *"back"*.) You can load a custom icon or pick a predefined glyph from [one of supported image sources](/Documentation/Guide/Themes_and_Styles/Icons/).  The sample below shows how to display an image from the [DevExtreme icon library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library):
```

Reply:
1)
The Icons article referenced in this paragraph defines standard and custom library icons, svg icons and images as icons. And "You can load a custom icon" can be applied to any of them. Also, there is a paragraph titled "Custom Images as Icons" - hence, there's a distinct "image" type of icons. Considering all these, can the paragraph be:

The [icon]() property adds a glyph to the button. (This property is not in effect if a button's type is *"back"*.) **You can load an icon from [one of supported sources]()**.  The sample below shows how to display an **glyph** from the [DevExtreme library]():

?

2) Why the first two sentences when they can be joined: "The [icon]() property adds a glyph to the button (this property is not in effect if a button's type is *"back"*)."?

Reply:
I can understand that you want to stay consistent with your terminology and that's good in general... But here we have a few sentences where we are forced to repeat the same word and I guess synonyms are a good trade-off. The way I've come to see these things is that our users don't necessarily learn/remember our terms by heart and it probably makes no difference to them if we say "image" or "icon" or "glyph". They are not going to remember that the fact that in our world "icons" and "images" are not the same. They have a lot of things going on in their lives, they read documentation from different sources on daily basis. I doubt they'll care enough to learn such a subtle difference. 

Anyway, your suggestion is fine. Although, my ideal version would still have separation between "load a custom image" and "load an image from a supported source". If we only say "supported sources", a reader may perceive it as a limitation. That is, we support a limited number of sources and you can only use images from those. But if we say "any custom icon or an image from our supported sources", then we imply freedom of choice (I can load anything) and simplicity (we also give you ready-to-use images). So maybe I'd go with something like this:

> You can load a custom icon or display one of the ready-to-use images from multiple supported image sources. 

(And yes, I understand that "load a custom image" is a "type of source" in that referenced document. But I'd rather write in such a way that doesn't require a reader to read and understand one more document.)

As for the second point - either way is okay of course. But given the choice between one longer sentence and two shorter sentences, I would usually go with two shorter sentences. I would only opt for a longer sentence in case it helps me avoid repetition and ends up actually being shorter than two sentences, which is not the case here. 

On a related note, I'm not even sure that this tiny detail (about the "back" button) is needed here in the Getting Started document. I was going to suggest that you remove it completely. There should probably be a note about it in API anyway. And if something changes, and there's another type that doesn't support custom icons, then you'll only have to update one topic, and won't have to update the Getting Started as well.